server/src/main/java/org/opensearch/snapshots/RestoreService.java

```diff
@@ -533,8 +543,18 @@ public ClusterState execute(ClusterState currentState) {
                                             indexMdBuilder.putAlias(alias);
                                         }
                                     } else {
-                                        for (final String alias : snapshotIndexMetadata.getAliases().keySet()) {
-                                            aliases.add(alias);
+                                        for (final Map.Entry<String, AliasMetadata> alias : snapshotIndexMetadata.getAliases().entrySet()) {
```

Comment:
Given we're repeating this logic from above, it might be best refactored into its own private method.

Reply:
Okay, done.